### DISCUSSION TITLE
cannot import name 'soft_unicode' from 'markupsafe' (U:\Python\Python310\lib\site-packages\markupsafe\__init__.py)

### DISCUSSION THREAD

**Discussion by NSH531** [pallets/flask#4822] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/5903577/190971598-4c727e5d-b40d-4bab-87ce-becdf33add74.png)

tnx
nate

**Comment by ThiefMaster** [comment]
https://github.com/pallets/flask/issues/4456#issuecomment-1044530631

**Comment by NSH531** [comment, marked as answer]
I'm reinstalling  flask and markupsafe
![image](https://user-images.githubusercontent.com/5903577/190973626-c173acc0-8db8-4191-a37a-e456b8472f38.png)

**Comment by ThiefMaster** [comment]
Look at the output. You didn't run the last command yet. And right now you are using the latest markupsafe with an older Flask version...

PS: Please stop posting screenshots for things that are just text.

**Comment by NSH531** [comment]
solved 
tnx
